api/apps/api/src/modules/scenarios/marxan-run/run.handler.ts

```diff
@@ -34,7 +34,7 @@ export class RunHandler {
   ) {}
 
   async runCalibration(scenarioId: string, blmValues: number[]) {
-    const assets = await this.assets.forScenario(scenarioId, 0);
+    const assets = await this.assets.forScenario(scenarioId, 1);
```

Comment:
Would be great to name the magic number for our future-selves

Reply:
I've created a variable with a descriptive name and a comment explaining the need of that particular value